cnew.py

```diff
@@ -124,9 +124,16 @@ def mainfunc(username, password, chatroom, servaddr):
         else:
             if chekroom(chatroom) is True:
                 print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <green>A valid chatroom identity was entered</green>"))
+            elif not chatroom.isupper():
+                chatroom = chatroom.upper()
+                if chekroom(chatroom):
+                    print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <green>A valid chatroom identity was entered</green>"))
+                else:
+                    print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <green>An invalid chatroom identity was entered</green>"))
+                    sys.exit()
             else:
-                print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <green>An invalid chatroom identity was entered</green>"))
-                sys.exit()
+                    print_formatted_text(HTML("[" + obtntime() + "] " + "SNCTRYZERO > <green>An invalid chatroom identity was entered</green>"))
+                    sys.exit()
```

Comment:
The `An invalid chatroom identity was entered` text should be in RED color.

Reply:
Done with the changes 👍